solr/core/src/java/org/apache/solr/cloud/overseer/NodeMutator.java

```diff
@@ -59,6 +57,8 @@ public List<ZkWriteCommand> downNode(ClusterState clusterState, ZkNodeProps mess
 
       String collection = entry.getKey();
       DocCollection docCollection = entry.getValue();
+      // PRS down state is already handled in ZkController.publishNodeAsDown(), so overseer can skip PRS collections
+      if (docCollection.isPerReplicaState()) continue;
```

Comment:
We may want throw runtimeException, if we ever reach here.

Reply:
I think we can still reach here, since we could potentially have a mix of PRS and non-PRS collections on a node. This is here so that if we do, we just skip PRS collections and only process the non-PRS collections.